src/components/Card.js

```diff
@@ -6,6 +6,13 @@ import './style.css';
 export const Card = () => {
   const firstLoad = useRef(true);
   const [qrvalue, setQrvalue] = useState('');
+  const smallSize = 175;
+  const mediumSize = 225;
+  const largeSize = 275;
+  const [size, setSize] = useState({
+    size: mediumSize,
+  });
+  var currentSize = size.size;
```

Comment:
Instead of adding this variable, you can just reference `size.size` wherever you need the current size.

Reply:
Done. I thought it would be more optimal - I mean we read a variable from an object once and write it to a variable, instead of querying the object for a variable each time.